peer/core.yaml

```diff
@@ -314,6 +314,14 @@ chaincode:
     # the image
     installpath: /opt/gopath/bin/
 
+    #keepalive in seconds. In situations where the communiction goes through a 
+    #proxy that does not support keep-alive, this parameter will maintain connection
+    #between peer and chaincode.
+    #A value <= 0 turns keepalive off
```

Comment:
Could you add a space after the `#` characters so it is consistent with the other comments in core.yaml?